Bottom: 5b15f5f7f4b37aba2e62a8a857486c6e41f1375f
Top:    2a851356dedaf0d611560fc342b239ed48b73bbd
Author: Parth Shah <parth@linux.ibm.com>
Date:   2020-01-09 18:41:07 +0530

Refresh of sched-core-add-manual

---

diff --git a/kernel/sched/sched.h b/kernel/sched/sched.h
index 0bda380d2845..71e2b19de51b 100644
--- a/kernel/sched/sched.h
+++ b/kernel/sched/sched.h
@@ -2499,3 +2499,5 @@ static inline void membarrier_switch_mm(struct rq *rq,
 #endif
 
 #define lenient_latency(lat)	((lat) == MAX_LATENCY_TOLERANCE)
+#define task_latency_lenient(lat)	\
+	(lenient_latency((p)->latency_tolerance))
